README.md

```diff
@@ -91,6 +92,31 @@ is provided by code that been generated from a model of the service.
 * [Contributions Guidelines](./documents/CONTRIBUTING.md)
 * [DEVELOPING](./documents/DEVELOPING.md)
 
+## FIPS Compliance
+
+This SDK supports [FIPS compliance](https://aws.amazon.com/compliance/fips/) on *some platforms*. If you're on a FIPS-compliant
+platform, FIPS will be enabled by default. See the [Platform-Specific JARs section](https://github.com/awslabs/aws-crt-java?tab=readme-ov-file#platform-specific-jars)
+in the CRT project for the current list of FIPS-compliant platforms.
+
+To interact with AWS IoT in a FIPS-compliant manner, you must use a [FIPS compliant endpoint](https://docs.aws.amazon.com/iot/latest/developerguide/iot-connect-fips.html#iot-connect-fips-data).
+
+> [!WARNING]
+> The Java IoT SDK can successfully connect to FIPS compliant endpoints on **non-FIPS-compliant** platforms.
+> Use `CRT.isFIPS()` call at runtime to ensure you are on a FIPS compliant platform.
+
+### Disabling FIPS compliance support
+
+If you're on a platform that supports FIPS, but you want to disable it, you need to remove `<classifier>fips-where-available</classifier>` line from the CRT dependency in `sdk/pom.xml`.
```

Comment:
I am not following this part.
Do we suggest user to modify the pom from iot-device-sdk and then build iot-device-sdk from source? That seems tricky

Reply:
Also, worth to think about if there are two different version/classifier of aws-crt within the dependency tree for the user, it really depend on maven and the order in the dependency tree to pick which one to use. (Arguably, it's up to the user to setup the right order to pick the expected one)

Reply:
You're right, it seems FIPS compliance support doesn't affect users that don't need it. I removed this section.
As for the two different versions of aws-crt: if the user calls `CRT.isFIPS()`, they should catch this situation.